### PR TITLE
[DOCFIX] Add some sentence in docs/cn/Running-Alluxio-on-EC2-Yarn.md

### PR DESCRIPTION
**Pull request by zhongtao93**

### CODE REVIEW

File: docs/cn/Running-Alluxio-on-EC2-Yarn.md [hunk 1]
```diff
@@ -116,7 +116,9 @@ Alluxio master/worker读取,将`~/.alluxio/`下的`alluxio-site.properties`放
 3. 运行Alluxio Master的节点的YARN的名称。（选填项，默认为`ALLUXIO_MASTER_HOSTNAME`）
 
 
-举例而言，启动3个worker节点的Alluxio集群，HDFS临时目录是`hdfs://AlluxioMaster:9000/tmp/`
+举例而言，启动3个worker节点的Alluxio集群，HDFS临时目录是`hdfs://AlluxioMaster:9000/tmp/`,主机的名字是`AlluxioMaster`
+
+你也可以分别从Yarn开启Alluxio Maswter节点，在这种情况下，上述启动将自动检测所在地址中所提供Master并跳过新实例的初始化。这是非常有用的，如果你有一个特定的主机，想在Master上运行，而这个主机不是你的Yarn集群的一部分，比如一个AWS EMR Master Instance。
```
Comment (saltylin) [inline docs/cn/Running-Alluxio-on-EC2-Yarn.md]: I think it is "Master", not "Maswter", right?

**Comment by saltylin** [comment]
@zhongtao93 Thanks very much for your translation. I've left a minor comment.

File: docs/cn/Running-Alluxio-on-EC2-Yarn.md (hunk 1 above)
Comment (zhongtao93) [inline docs/cn/Running-Alluxio-on-EC2-Yarn.md]: Yes,it is "Master", I'm sorry to make such a silly mistake.

File: docs/cn/Running-Alluxio-on-EC2-Yarn.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Alluxio Web UI的默认端口为**19999**。
 
 Hadoop Web UI的默认端口为**50070**。
 
-在浏览器中输入`http://{MASTER_IP}:{PORT}`地址访问Web UI。
+在浏览器中输入`http://{MASTER_IP}:{PORT}`地址访问Web UIs。
```
Comment (apc999) [inline docs/cn/Running-Alluxio-on-EC2-Yarn.md]: is it necessary to change UI to UIs in Chinese?

Comment (zhongtao93) [inline docs/cn/Running-Alluxio-on-EC2-Yarn.md]: Sorry, UI is right.

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Pull request merged**